index.html

```diff
@@ -15,10 +17,9 @@
   <h1 id="header"> Amazon Trip</h1>
   <div class="traveler">
     <div id="profile">
-      <img src="https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg"
-        alt="{traveler name}">
-      <h2>{traveler Name}</h2>
-      <em>{traveler Nickname}</em>
+      <img src="image.textContent" alt="name.textContent">
+      <h2>name.textContent</h2>
+      <em>nickName.textContent</em>
```

Comment:
No need to update the HTML manually here - our JS code will take care of that :)